### PR TITLE
Move to pytest-workflow

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
-  then git submodule foreach --recursive bash -c
-  'if [ "$(git tag --contains)" == "" ] ;  then git checkout develop && git pull && git submodule update --init --recursive ; else echo "on tag: $(git tag --contains)" ; fi' ;
-  fi
-- "git diff --exit-code || (echo ERROR: Git changes detected. Submodules should either be tagged or on the latest version of develop. && exit 1)"
+  - python3 -m pytest --tag $TEST
+
+matrix:
+  include:
+    - env: TEST=guided
+    - env: TEST=unguided
```
Comment (DavyCats) [inline .travis.yml:37]: The `GuidedWithNovel` test is missing

File: tests/test_bammetrics.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+- name: guided
```
Comment (DavyCats) [inline tests/test_bammetrics.yml]: I think you forgot to rename this file.

**Review by DavyCats — CHANGES_REQUESTED** [review]

File: tests/integration/guided_with_novel.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "MultiBamExpressionQuantification.strandedness": "None",
+  "MultiBamExpressionQuantification.referenceGtfFile": "tests/data/reference.gtf",
```
Comment (DavyCats) [inline tests/integration/guided_with_novel.json:3]: This is missing one input:
```suggestion
  "MultiBamExpressionQuantification.referenceGtfFile": "tests/data/reference.gtf",
  "MultiBamExpressionQuantification.detectNovelTranscripts": true,
```

**Review by DavyCats — CHANGES_REQUESTED** [review]

File: tests/integration/guided_with_novel.json (hunk 3 above)
Comment (rhpvorderman) [reply tests/integration/guided_with_novel.json:3]: Whoops must have been a bit distracted.

**Review by DavyCats — APPROVED** [review]

**Pull request merged**